CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 13.2.2 - 2019-02-20
+### Changed
```

Comment:
I think this should be `Added` rather than `Changed`
and then you can remove the word 'Added' from the start of statement.

https://keepachangelog.com/en/1.0.0/

Reply:
Thanks for the link, looks interesting. Will update now.